website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -426,3 +426,5 @@ Managed Kubernetes Clusters can be imported using the `resource id`, e.g.
 ```shell
 terraform import azurerm_kubernetes_cluster.cluster1 /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/group1/providers/Microsoft.ContainerService/managedClusters/cluster1
```

Comment:
Should we just update the import documentation instead?

Reply:
If could just update import command on document I think it will be better. Thanks for the advise.